server/lib/nl/detection/place.py

```diff
@@ -66,7 +70,7 @@ def detect_from_query_dc(orig_query: str,
           d['dcid'] for d in item['places'] if 'dcid' in d
       ]
     else:
-      nonplace_query_parts.append(item['span'].lower())
+      query_parts_with_dcids.append(item['span'].lower())
```

Comment:
Add comment on what this else means?  Is the span missing DCID vs. non-place thing?

Reply:
done